mizar/common/rpc.py

```diff
@@ -380,7 +390,17 @@ def delete_network_policy_ingress(self, cidr_networkpolicy_list):
                 "cidr_type": cidr_networkpolicy.get_cidr_type_int(),
                 "bit_value": str(cidr_networkpolicy.policy_bit_value),
             }
-            conf_list.append(conf)
+            # +1 is for comma that will get added during json conversion
```

Comment:
this block of code aplitting list to batches not exceeding a certian number, seems to repeat quite some times. Is it possible to reduce code repetition and insease code reuse?